django-stubs/db/models/functions/__init__.pyi

```diff
@@ -88,3 +88,103 @@ from .window import Ntile as Ntile
 from .window import PercentRank as PercentRank
 from .window import Rank as Rank
 from .window import RowNumber as RowNumber
+
+__all__ = [
+    # comparison and conversion
+    "Cast",
+    "Coalesce",
+    "Collate",
+    "Greatest",
+    "Least",
+    "NullIf",
+    # datetime
+    "Extract",
+    "ExtractDay",
+    "ExtractHour",
+    "ExtractMinute",
+    "ExtractMonth",
+    "ExtractQuarter",
+    "ExtractSecond",
+    "ExtractWeek",
+    "ExtractIsoWeekDay",
+    "ExtractWeekDay",
+    "ExtractIsoYear",
+    "ExtractYear",
+    "Now",
+    "Trunc",
+    "TruncDate",
+    "TruncDay",
+    "TruncHour",
+    "TruncMinute",
+    "TruncMonth",
+    "TruncQuarter",
+    "TruncSecond",
+    "TruncTime",
+    "TruncWeek",
+    "TruncYear",
+    # json
+    "JSONArray",
```

Comment:
we need to add this object: https://github.com/django/django/blob/0596263c3136bc26cffa670e5322bd0aa56c4d34/django/db/models/functions/json.py#L8